Tests/SentryTests/Integrations/ANR/SentryANRTypeTests.swift

```diff
@@ -0,0 +1,98 @@
+@testable import Sentry
+import XCTest
+
+final class SentryANRTypeTests: XCTestCase {
+
+   func testExceptionType_FatalFullyBlocking() {
+      XCTAssertEqual(
```

Comment:
`l`: nit pick, i believe in many (maybe even most) other test assertions we are going for a `XCTAssertEqual(<VALUE>, <EXPECTED VALUE>)` approach, here it's the other way around

Reply:
I usually do `XCTAssertEqual(<EXPECTED VALUE>, <VALUE>)`. The other tests already had `XCTAssertEqual(<VALUE>, <EXPECTED VALUE>)`, so I stuck to it there.

Reply:
I usually prefer `XCTAssertEqual(<VALUE>, <EXPECTED VALUE>)`

Reply:
I took the discussion to Slack: https://sentry.slack.com/archives/C040TTUKQT1/p1740750007985469.